src/list.ts

```diff
@@ -0,0 +1,259 @@
+            anzahl: Number.parseInt(quantity.value),
+            erledigt: false,
+            einkaufslisten: list.$id!
+        }
+
+        database.createDocument(
+            CONFIG.DATABASE_ID,
+            CONFIG.DB_COLLECTION_SHOPLISTENTRY,
+            ID.unique(),
+            product,
+            [
+                Permission.read(Role.team(list.ID_Household)),
+                Permission.update(Role.team(list.ID_Household)),
+                Permission.delete(Role.team(list.ID_Household)),
+            ],
+        ).then(() => { },
+            () => { window.alert('Das geforderte Produkt konnte nicht erstellt werden. :(') });
+    });
+
+    list.listeneintrag.forEach(function (value: Listeneintrag) {
+        productTable.appendChild(createTableRow(value));
+    });
+}, () => {
+    window.location.pathname = '/uebersicht.html';
+});
+
+function createTableRow(data: Listeneintrag, id?: string): HTMLTableRowElement {
+    const row = document.createElement('tr');
+    if (id) {
+        row.setAttribute('id', id!);
+    } else {
+        row.setAttribute('id', data.$id!);
+    }
+
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.checked = data.erledigt;
+    checkbox.addEventListener('change', async () => {
+        data.erledigt = Boolean(checkbox.checked);
+        await updateListEntry(data);
+    });
+
+    const productName = document.createElement('input');
+    productName.type = 'text';
+    productName.disabled = true;
+    productName.value = data.artikelname;
+
+    const quantity = document.createElement('input');
+    quantity.type = 'number';
+    quantity.min = '1';
+    quantity.max = '999';
+    quantity.value = String(data.anzahl);
+    quantity.disabled = true;
+
+    const editButton = document.createElement('button');
+    editButton.innerHTML = '&#9998';
+
+    const deleteButton = document.createElement('button');
+    deleteButton.innerHTML = '&#x1F5D1;';
+
+    deleteButton.addEventListener('click', async (event) => {
+        event.preventDefault();
+
+        database.deleteDocument(
+            CONFIG.DATABASE_ID,
+            CONFIG.DB_COLLECTION_SHOPLISTENTRY,
+            data.$id!
+        ).then(() => { },
+            () => {
+                window.alert('Das ausgewählte Produkt konnte nicht gelöscht werden. :(');
+            });
+    });
+
+    editButton.addEventListener('click', async (event) => {
+        event.preventDefault();
+        if (!productName.disabled.valueOf()) {
+            data.anzahl = Number.parseInt(quantity.value);
+            data.artikelname = productName.value;
+            await updateListEntry(data);
+        }
+        productName.disabled = !productName.disabled.valueOf();
+        quantity.disabled = !quantity.disabled.valueOf();
+    });
+
+    addToRow(checkbox, row);
+    addToRow(productName, row);
+    addToRow(quantity, row);
+    addToRow(editButton, row);
+    addToRow(deleteButton, row);
```

Comment:
A bit of "Don't repeat yourself" might be warranted here. a "for of" loop with an array containing the `HTMLElement`s would do.